tin-drink/src/components/content/showDrink/ShowDrink.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import './ShowDrink.css'
+
+class ShowDrink extends React.Component {
+
+    state = {
+        selectedIngredient: "",
+        drinkName: "",
+        drinkTaste: "",
+        drinkDesc: "",
+        drinkVol: "",
+        drinkId:"",
+        photoFile: "",
+        ingredientAmount: "",
+        drinkNameError:"",
+        selectedIngredientError: "",
+        ingredientAmountError: "",
+        numberOfIngredientsError: "",
+        addDrinkError: ""
+
+    }
+
+    handleBackButton = () => {
+        this.props.history.push({
+            pathname: '/Lista_Drinkow',
+        })
+    }
+
+    componentDidMount() {
+        if(this.props.location.id){
+          fetch(`http://localhost:8080/api/drinks/${this.props.location.id}`)
+          .then(res => res.json())
+          .then(data => {
+              console.log(data)
+              this.setState({
+                  selectedIngredientsList: data.amounts,
+                  drinkName: data.name,
+                  drinkTaste: data.taste,
+                  drinkDesc: data.drinkDesc,
+                  drinkVol: data.vol,
+                  photoPath: data.photoPath
+
+              })
+          });
+        }
+    }
+
+    render () {
+        return (
+            <>
+            <div id="single-drink" className="show-single-drink">
+                <div id="drink-photo" class="drink-photo">
+                    <img src={`http://localhost:8080/images/${this.state.photoPath}`} onClick={(e) => this.showDrink()}/>
```

Comment:
img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.